config/locales/simple_form.en.yml

```diff
@@ -0,0 +1,31 @@
+en:
```

Comment:
You should probably add russian language file with proper localization to this folder.

Reply:
I will add this in future, when I will add more functionality